satpy/readers/utils.py

```diff
@@ -189,3 +189,19 @@ def unzip_file(filename):
         return tmpfilepath
 
     return None
+
+
+def bbox(img):
```

Comment:
I wouldn't be surprised if something like this already exists in pyresample (@mraspaud ?).